src/app/api/cycles/route.ts

```diff
@@ -0,0 +1,43 @@
+import { NextResponse } from 'next/server'
+import { db } from '@/lib/db'
+import { paginateQuery } from '@/app/api/utils/paginateQuery'
+import { parsePaginationParams } from '@/app/api/utils/parsePaginationParams'
+
+const COLUMNS = [
```

Comment:
NIT: I think it would be useful to have a db folder with a constants file inside where we define the tables and the columns that we're using. WDYT?

Reply:
Maybe we can discuss this to be on the same page :)